### DISCUSSION TITLE
Custom Langserve server for chat-ui

### DISCUSSION THREAD

**Discussion by EswarSk** [huggingface/chat-ui#1231] [Q&A]
Did anyone create a rag application with langserve that is compatible with the chat-ui? I want to create an endpoint that will work smoothly with the UI but stuck with the streaming part. I'm looking for any reference that can help. Thanks
`const r = await fetch(`${url}/stream`, {
			method: "POST",
			headers: {
				"Content-Type": "application/json",
			},
			body: JSON.stringify({
				input: { text: prompt },
			}),
		});`

**Comment by adhishthite** [comment]
Hey, let me know what you need.
